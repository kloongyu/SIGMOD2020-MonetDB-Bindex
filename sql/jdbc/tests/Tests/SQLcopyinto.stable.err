stderr of test 'SQLcopyinto` in directory 'sql/jdbc/tests` itself:


# 12:51:59 >  
# 12:51:59 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33746" "--set" "mapi_usock=/var/tmp/mtest-12494/.s.monetdb.33746" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/dinther/INSTALL/var/MonetDB/mTests_sql_jdbc_tests" "--set" "embedded_r=yes" "--set" "embedded_py=true" "--set" "embedded_c=true"
# 12:51:59 >  

# builtin opt 	gdk_dbpath = /export/scratch1/dinther/INSTALL/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 33746
# cmdline opt 	mapi_usock = /var/tmp/mtest-12494/.s.monetdb.33746
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /export/scratch1/dinther/INSTALL/var/MonetDB/mTests_sql_jdbc_tests
# cmdline opt 	embedded_r = yes
# cmdline opt 	embedded_py = true
# cmdline opt 	embedded_c = true
# cmdline opt 	gdk_debug = 553648138

# 12:52:00 >  
# 12:52:00 >  "./SQLcopyinto.SQL.sh" "SQLcopyinto"
# 12:52:00 >  


# 12:52:00 >  
# 12:52:00 >  java SQLcopyinto "jdbc:monetdb://catskill:33746/mTests_sql_jdbc_tests?user=monetdb&password=monetdb"
# 12:52:00 >  


# 12:52:00 >  
# 12:52:00 >  "Done."
# 12:52:00 >  

