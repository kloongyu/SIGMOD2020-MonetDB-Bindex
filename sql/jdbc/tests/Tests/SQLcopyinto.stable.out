stdout of test 'SQLcopyinto` in directory 'sql/jdbc/tests` itself:


# 12:51:59 >  
# 12:51:59 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33746" "--set" "mapi_usock=/var/tmp/mtest-12494/.s.monetdb.33746" "--set" "monet_prompt=" "--forcemito" "--dbpath=/export/scratch1/dinther/INSTALL/var/MonetDB/mTests_sql_jdbc_tests" "--set" "embedded_r=yes" "--set" "embedded_py=true" "--set" "embedded_c=true"
# 12:51:59 >  

# MonetDB 5 server v11.29.4
# This is an unreleased version
# Serving database 'mTests_sql_jdbc_tests', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 31.307 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2018 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://catskill.da.cwi.nl:33746/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-12494/.s.monetdb.33746
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
# MonetDB/Python2 module loaded
# MonetDB/R   module loaded

Ready.

# 12:52:00 >  
# 12:52:00 >  "./SQLcopyinto.SQL.sh" "SQLcopyinto"
# 12:52:00 >  


# 12:52:00 >  
# 12:52:00 >  java SQLcopyinto "jdbc:monetdb://catskill:33746/mTests_sql_jdbc_tests?user=monetdb&password=monetdb"
# 12:52:00 >  

SQLcopyinto started
Connected to MonetDB server

CopyInto STDIN begin
Before connecting to MonetDB server via MapiSocket
Connected to MonetDB server via MapiSocket
Before sending data to STDIN
Completed sending data via STDIN
CopyInto STDIN end

Listing uploaded data:
Row data: 0, val_0
Row data: 1, val_1
Row data: 2, val_2
Row data: 3, val_3
Row data: 4, val_4
Row data: 5, val_5
Row data: 6, val_6
Row data: 7, val_7
Row data: 8, val_8
Row data: 9, val_9
Row data: 10, val_10
Row data: 11, val_11
Row data: 12, val_12
Row data: 13, val_13
Row data: 14, val_14
Row data: 15, val_15
Row data: 16, val_16
Row data: 17, val_17
Row data: 18, val_18
Row data: 19, val_19
Row data: 20, val_20
Row data: 21, val_21
Row data: 22, val_22
Row data: 23, val_23
Row data: 24, val_24
Row data: 25, val_25
Row data: 26, val_26
Row data: 27, val_27
Row data: 28, val_28
Row data: 29, val_29
Row data: 30, val_30
Row data: 31, val_31
Row data: 32, val_32
Row data: 33, val_33
Row data: 34, val_34
Row data: 35, val_35
Row data: 36, val_36
Row data: 37, val_37
Row data: 38, val_38
Row data: 39, val_39
SQLcopyinto completed

# 12:52:00 >  
# 12:52:00 >  "Done."
# 12:52:00 >  

